### ISSUE TITLE
 test_create_detection_from_ms_small and test_create_detection_from_ms are deterministic

### ISSUE THREAD

**Issue by deiruch** [i4Ds/Karabo-Pipeline#187]
`SkyModel.get_random_poisson_disk_sky` should be seeded.

**Comment by Lukas113** [comment]
I've seen that we were using a random generator from `lib/python<version>/random.py` in `karabo/util/math_util.py` `poisson_disc_samples`. I think we should have all random generators of our own code be `numpy` generators so an easy `np.random.seed(number)` seeds everything. I haven't found any other occurences of non `numpy` random generators in the repo. But I've just searched for *import random* in our repository.

I've replaced these random generators using `numpy.random.rand`
